.travis.yml

```diff
@@ -7,6 +7,11 @@ os: osx
 osx_image: xcode6.4
 
 env:
+  matrix:
+    - CONFIG=osx_python2.7
+    - CONFIG=osx_python3.5
+    - CONFIG=osx_python3.6
```

Comment:
All recipes that are using VCs and `python` to activate them should not be re-rendered until we fix them to the new format. Closing this for now.